src/server/campaignd/server.cpp

```diff
@@ -1426,8 +1452,15 @@ void server::handle_upload(const server::request& req)
 	addon["uploads"] = 1 + addon["uploads"].to_int();
 
 	addon.clear_children("feedback");
+	int topic_id = 0;
 	if(const config& url_params = upload.child("feedback")) {
 		addon.add_child("feedback", url_params);
+		// already validated that this can be converted to an int in validate_addon()
+		topic_id = url_params["topic_id"].to_int();
+	}
+
+	if(user_handler_) {
+		user_handler_->db_insert_addon_info(server_id_, name, addon["title"].str(), addon["type"].str(), addon["version"].str(), false, topic_id);
```

Comment:
Ehhh, I didn't really add any code to ensure `server_id_` isn't empty. In fact, the client expects it to potentially be empty/not present. What do you think should be done about this?

Reply:
What's the use case for it being empty?

Reply:
Not having to set any configuration when booting up a throwaway test instance.

Reply:
Then I'd say the solution is to log an error and then exit on startup if database support is enabled but the ID is missing.

Reply:
Then you know what to do. :p

Reply:
Alright, updated.